component/src/test/java/io/siddhi/extension/io/nats/sink/NATSSinkTestCase.java

```diff
@@ -305,123 +135,10 @@ public void testIncorrectNatsServerUrl() throws InterruptedException {
         SiddhiAppRuntime executionPlanRuntime = siddhiManager.createSiddhiAppRuntime(siddhiApp);
         executionPlanRuntime.start();
         Thread.sleep(500);
-        Assert.assertTrue(appender.getMessages().contains("Error while connecting to NATS server at destination:"));
+        Assert.assertTrue(appender.getMessages().contains("Error in Siddhi App Test-plan8 while connecting to NATS " +
+                "server endpoint [nats://localhost:5223]"));
         siddhiManager.shutdown();
     }
 
-    /**
-     * If the map annotation is not included in the sink annotation then {@link SiddhiAppCreationException} should be
-     * thrown.
-     */
-    @Test(expectedExceptions = SiddhiAppCreationException.class, dependsOnMethods = "testIncorrectNatsServerUrl")
```

Comment:
Any reason to remove these testcases?

Reply:
These testcases are added in STANSinkTestCase.java class, since these are nats streaming testcases.

Reply:
Ack